qa/common/deploy.sh

```diff
@@ -118,17 +116,22 @@ function _zypper_ps {
     salt '*' cmd.run 'zypper ps -s' 2>/dev/null || true
 }
 
+function _python_versions {
+    type python2 > /dev/null 2>&1 && python2 --version || echo "Python 2 not installed"
+    type python3 > /dev/null 2>&1 && python3 --version || echo "Python 3 not installed"
+}
+
 function initialization_sequence {
     set +x
```

Comment:
@kshtsk `_initialize_minion_array` is called from this function (`initialization_sequence`), and `set +x` is in effect for the entire function. That's why I dropped it from `_initialize_minion_array`.

I suggest we try this and see if the logs are more readable/less confusing.